models/rollup_economics/ethereum/rollup_economics_ethereum_l1_fees.sql

```diff
@@ -0,0 +1,136 @@
+{{ config(
+    alias = alias('l1_fees'),
+    tags = ['dunesql'],
+    partition_by = ['block_time'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['tx_hash'],
+    post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "project",
+                                    "rollup_economics",
+                                    \'["niftytable"]\') }}'
+)}}
+
+WITH tx_batch_appends AS (
+    SELECT
+    'Arbitrum' AS name,
```

Comment:
minor: can you make lowercase? that's the trend across most of spellbook

Reply:
done